packages/expo-video/src/VideoPlayer.types.ts

```diff
@@ -171,20 +171,33 @@ export type VideoPlayerStatus = 'idle' | 'loading' | 'readyToPlay' | 'error';
 
 export type VideoSource =
   | string
+  | number
   | {
       /**
        * The URI of the video.
+       *
+       * This property is exclusive with the `assetId` property, when both are present the `assetId` will be ignored.
```

Comment:
```suggestion
       * This property is exclusive with the `assetId` property. When both are present, the `assetId` will be ignored.
```

Let's split the sentence here.